index.html

```diff
@@ -33,6 +33,293 @@
+      <a data-cite="screen-capture#dfn-display-surface">display surface</a> in the
+      form of a video track.
+    </p>
+    <p>
+
+      Users currently cannot capture multiple
+      <a data-cite="screen-capture#dfn-monitor">monitors</a>
+      at once without a [=transient activation=]. This may be useful in certain
+      scenarios, e.g. for compliance or training purposes.
+      The only currently available alternative is through the Screen Capture API with
+      <a data-cite="screen-capture#dom-mediadevices-getdisplaymedia">getDisplayMedia</a>
+      by calling
+      <a data-cite="screen-capture#dom-mediadevices-getdisplaymedia">getDisplayMedia</a>
+      several times and asking the user to select the available
+      <a data-cite="screen-capture#dfn-monitor">monitors</a>.
+      This requires multiple user gestures, repeated interaction with browser-level UX.
+      This adds unnecessary friction and results in degraded user experience and
+      interrupted workflows.
+    </p>
+    <p>
+      This document describes {{MediaDevices/getAllScreenMedia}}, an extension to
+      the Screen Capture API [[screen-capture]].
+      {{MediaDevices/getAllScreenMedia}} enables
+      the capturing of several of the user's <a data-cite="screen-capture#dfn-monitor">monitors</a>,
+      or parts thereof, without a [=transient activation=].
+      It returns a list of media streams
+      (each containing a track corresponding to the captured
+      <a data-cite="screen-capture#dfn-monitor">monitor</a>).
+      As no [=transient activation=] is required, the <a href="#dfn-user-agent">user agent</a>
+      MUST ensure that appropriate protection, e.g. by allowlisting permitted
+      <a data-cite="url#concept-url-origin">origin</a>.
+    </p>
+  </section>
+  <section id="conformance">
+    <p>
+      This specification defines conformance criteria that apply to a single product: the
+      <dfn data-lt="user-agent">user agent</dfn> that implements the interfaces that it
+      contains.
+    </p>
+    <p>
+      Implementations that use ECMAScript [[ECMA-262]] to implement the APIs defined in this
+      specification must implement them in a manner consistent with the ECMAScript Bindings
+      defined in the Web IDL specification [[!WEBIDL]], as this specification uses that
+      specification and terminology.
+    </p>
+  </section>
+  <section>
+    <h2>
+      Example
+    </h2>
+    <p>
+      The following example demonstrates a request for all
+      <a data-cite="screen-capture#dfn-monitor">monitors</a>
+      using the
+      <code>navigator.mediaDevices.getAllScreenMedia</code>
+      method defined in this document.
+    </p>
+    <pre class="example highlight">
+      try {
+        const mediaStreams = await navigator.mediaDevices.getAllScreenMedia();
+        mediaStreams.forEach((mediaStream, index) => {
+          videoElements[index].srcObject = mediaStream;
```

Comment:
Subjective: `files.push(saveToFile(mediaStream))` might be more illustrative. Or less. Up to you.